README.md

```diff
@@ -1 +1,20 @@
-# openvino_quantization
\ No newline at end of file
+# openvino_quantization
+
+Link to find ResNet_V2_101 model:
+https://www.tensorflow.org/lite/guide/hosted_models
+Direct link to download this model:
+https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/resnet_v2_101.tgz
+
+To run demo_classification_tf.py:
+```bash
+python demo_classification_tf.py --graph resnet_v2_101_299_frozen.pb --image example.jpeg
+```
+
+To run evaluate_tf.py:
```

Comment:
Please change description. Otherwise "to run evaluate_tf.py you need to run evaluate_tf.py".

Something like "To validate TensorFlow model on ImageNet dataset run"

Reply:
the same for demo script